storage/src/qmdb/benches/fixed/init.rs

```diff
@@ -48,23 +48,20 @@ fn bench_fixed_init(c: &mut Criterion) {
                                 gen_random_kv(db, elements, operations, Some(COMMIT_FREQUENCY))
                                     .await;
                             db.prune(db.inactivity_floor_loc()).await.unwrap();
-                            db.close().await.unwrap();
```

Comment:
I didn't think the call to close is a meaningful part of the benchmark so I removed it